Makefile

```diff
@@ -119,7 +119,7 @@ dist-tar:
 	@rm -rf ${DIST}
 	@mkdir ${TMP}
 	@mkdir ${DIST}
-	@cp -r .git cmd bin conf docker docs grafana pkg service go.mod go.sum Makefile README.md LICENSE ${TMP}
+	@cp -r .git cmd bin conf docker docs grafana pkg service autosupport go.mod go.sum Makefile README.md LICENSE ${TMP}
```

Comment:
Also change asup target to keep or overwrite binary in correct folder

Reply:
changed binary path in asup target.